src/main/java/jenkins/plugins/logstash/LogstashInstallation.java

```diff
@@ -85,6 +90,9 @@ public boolean configure(StaplerRequest req, JSONObject formData) throws FormExc
     }
 
     @Override
+    @SuppressFBWarnings(
+      value="NP_PARAMETER_MUST_BE_NONNULL_BUT_MARKED_AS_NULLABLE",
+      justification="TODO: investigate")
     public ToolInstallation newInstance(StaplerRequest req, JSONObject formData) throws FormException {
```

Comment:
Req is marked as nullable in the interface, so apparently we should handle this gracefully.

Reply:
I just suppressed the warning, I don't think I touched anything about this. Good to go after all suppression later and have them fixed.